src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -413,7 +417,6 @@ object Gemini {
   val defaultBblfshPort: Int = 9432
   val defaultFeHost: String = "127.0.0.1"
   val defaultFePort: Int = 9001
-  val defaultDocFreqFile: String = "docfreq.json"
```

Comment:
I see big benefit in having all the configuration defaults in the single place, do you think it might be worth keeping it that way for now? Then we could get back to this single place and clean it up.

Reply:
makes sense. Fixed.